webapp-django/crashstats/crashstats/templates/crashstats/crashes_per_user.html

```diff
@@ -0,0 +1,275 @@
+                          <fieldset>
+                              <legend class="visually-hidden">Date Range Type</legend>
+                              <span class="radio-item">
+                                  <label for="by_build">
+                                      <input type="radio" name="date_range_type" id="by_build" value="build" {% if date_range_type == 'build' %} checked="checked" {% endif %}/> Build Date
+                                  </label>
+                              </span>
+                              <span class="radio-item">
+                                  <label for="crash">
+                                      <input type="radio" name="date_range_type" id="crash" value="report" {% if date_range_type == 'report' %} checked="checked" {% endif %}/> Crash Date
+                                  </label>
+                              </span>
+                            </fieldset>
+                          </td>
+                    </tr>
+                    #}
+
+                    <tr class="datepicker-daily">
+                        <th>Date</th>
+                        <td>
+                          <label class="visually-hidden" for="when_date_start">Start Date</label>
+                          <input class="date" id="when_date_start" type="text" name="date_start" value="{{ start_date }}" />
+                          &nbsp; to &nbsp;
+                          <label class="visually-hidden" for="when_date_end">End Date</label>
+                          <input class="date" id="when_date_end" type="text" name="date_end" value="{{ end_date }}" />
+                        </td>
+                    </tr>
+                </table>
+
+                <input id="daily_search_version_form_submit" type="submit" name="submit" value="Generate">
+            </form>
+        </div>
+        </div>
+    </div>
+</div>
+
+    <div class="panel daily_graph">
+        <div class="title">
+            <h2>Crashes per 100 ADIs</h2>
+        </div>
+
+        <div class="body">
+                <div id="crashes-per-adi-graph"></div>
+                <div id="crashes-per-adi-legend" class="crashes-per-adi-legend"></div>
+            <p class="adu-chart-help">This graph uses an approximate <a href="https://wiki.mozilla.org/Socorro/SocorroUI/Branches_Admin#Throttle">throttle value</a> for each version, which may not be completely accurate for the entire time period.</p>
+            </div>
+    </div>
+
+
+    <br class="clear" />
+
+    <div class="panel daily_">
+        <div class="title">
+            <h2>Crashes per ADI</h2>
+            <div class="choices">
+                <ul>
+                    {% if request.META.get('QUERY_STRING') %}
+                    <li><a href="?{{ request.META.get('QUERY_STRING') }}&amp;format=csv">csv</a></li>
+                    {% else %}
+                    <li><a href="?format=csv">csv</a></li>
+                    {% endif %}
+                </ul>
```

Comment:
Still works. And people are using it. I don't know how much. 

It wouldn't be easy to replace that with instructions how how to use the API because the output involves doing gets on the ADI, the ProductBuildTypes AND the SuperSearch.